crypto-ffi/bindings/js/src/CoreCryptoInstance.ts

```diff
@@ -367,32 +367,8 @@ export class CoreCrypto {
     }
 
     /**
-     * If this returns `false` you **cannot** call {@link CoreCrypto.close} as it will produce an error because of the
-     * outstanding references that were detected.
-     *
-     * As always with this kind of thing, beware TOCTOU.
-     *
-     * @returns whether the CoreCrypto instance can currently close.
-     */
-    async canClose(): Promise<boolean> {
-        return await this.#cc.can_close();
-    }
-
-    /**
-     * If this returns `true` you **cannot** call {@link CoreCrypto.close} as it will produce an error because of the
-     * outstanding references that were detected.
-     *
-     * This will never return `true` as we need an async method to accurately determine whether or not this can close.
-     *
-     * @returns false
-     * @deprecated prefer {@link CoreCrypto.canClose}
-     */
-    isLocked(): boolean {
-        return false;
-    }
-
-    /**
-     * Closes this {@link CoreCrypto} instance and deallocates all loaded resources
+     * Wait for any transaction that is currently in progress, then closes this {@link CoreCrypto}
```

Comment:
Nit: to be consistent in style, we should use either "Wait for..., then close ..." or "Waits for..., then closes ...".
Looking at other nearby functions, it seems the latter style is more common.